RFCs/RFC-001: Introspect streams.md

```diff
@@ -0,0 +1,62 @@
+# RFC-001: Introspect Streams
```

Comment:
How about naming this `minimal CLI support for writing and monitoring streams`

Reply:
changed to: "CLI support for publishing and subscribing to streams"